requirements/edx/github.txt

```diff
@@ -50,7 +50,7 @@ git+https://github.com/edx/edx-lint.git@8bf82a32ecb8598c415413df66f5232ab8d974e9
 -e git+https://github.com/edx/xblock-utils.git@581ed636c862b286002bb9a3724cc883570eb54c#egg=xblock-utils
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 -e git+https://github.com/edx/edx-reverification-block.git@2ff0d21f6614874067168bd244e68d8215041f3b#egg=edx-reverification-block
-git+https://github.com/edx/ecommerce-api-client.git@0.3.0#egg=ecommerce-api-client==0.3.0
+git+https://github.com/edx/ecommerce-api-client.git@tyler-jwt-fullname#egg=ecommerce-api-client==tyler-jwt-fullname
```

Comment:
We use semantic versioning to tag [releases of the ecommerce API client](https://github.com/edx/ecommerce-api-client/releases). The current version is `0.4.3`. Please tag your commit introducing full names to the JWTs generated by the client as `0.4.4`, and then use it here.